source/pymongo.txt

```diff
@@ -56,11 +54,11 @@ version of the module using the command line:
 
 If you need to install a specific version of PyMongo, specify the
 version in your command. The following command shows how you can use
-``pip`` to install PyMongo version ``3.11``:
+``pip`` to install PyMongo version ``4.3``:
```

Comment:
Suggestion: 
If this is something we need to keep up to date, use a source constant for the latest version and add that to the task checklist for docs-ecosystem.

Reply:
this isn't something that is updated to the most recent version, It seems like this was an arbitrary version selected to show users how to select a specific version to install. I thought maybe updating it to a new_er_ version made sense, but since it is arbitrary I'll just revert the change